projecthub/test/projecthub/api/grpc_server_test.exs

```diff
@@ -2964,6 +2964,80 @@ defmodule Projecthub.Api.GrpcServerTest do
     end
   end
 
+  describe ".restore" do
+    test "when a soft deleted project is set to be restored => restores the project and returns ok" do
+      {:ok, channel} =
+        GRPC.Stub.connect("localhost:50051",
+          interceptors: [
+            Projecthub.Util.GRPC.ClientRequestIdInterceptor,
```

Comment:
👋🏻 Unless you explicitly need them, these interceptors can be safely omitted in tests.